packages/components/pie-icon-button/test/visual/pie-icon-button.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { test } from '@sand4rt/experimental-ct-web';
```

Comment:
Sorry fairly late to this PR. I was looking adding tests in my PR and I initially struggled to work out want is going on in them if im honest. If it's feasible I think we should try and aim to clean these tests up a little so it's easier to follow, there might be some valid reasons why we need these helpers and the promise loops which is fine (or maybe it's a temp solution?) but if it means splitting helpers out and possibly not using loops for readability it might make them easier to follow. Open to suggestions too.